Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1999,6 +2036,27 @@ def send_draft_command(client: MsGraphClient, args):
     return_outputs(f'### Draft with: {draft_id} id was sent successfully.')
 
 
+def update_email_status_command(client: MsGraphClient, args) -> CommandResults:
+    user_id = args['user_id']
+    folder_id = args.get('folder_id')
+    message_ids = argToList(args['message_ids'])
+    status: str = args['status']
+    mark_as_read = (status.lower() == 'read')
+
+    raw_responses = []
+
+    for message_id in message_ids:
+        raw_responses.append(
+            client.update_email_read_status(user_id=user_id, message_id=message_id,
+                                            folder_id=folder_id, read=mark_as_read)
+        )
+
+    return CommandResults(
+        readable_output=f'Emails status has been updated to {status}.',
+        raw_response=raw_responses[0] if len(raw_responses) == 1 else raw_responses
```

Comment:
In case only a single email ID was passed to this function, send in `raw_response` the single response as it is (a dictionary), instead of returning it in a single-item-list.

Reply:
There is no need to manipulate API responses.
It's better to return in `raw_response` the API response as is.

Reply:
The function sends a separate API request in a loop for each email it's updating (it's not a single API request for updating multiple emails).

So if multiple emails are updated, I combine all of the responses (one for each email updated) to a list and return it.
If not, I just return the single response as it is.